src/Microsoft.DotNet.Cli.Utils/MSBuildForwardingAppWithoutLogging.cs

```diff
@@ -23,9 +23,12 @@ internal class MSBuildForwardingAppWithoutLogging
             new Dictionary<string, string>
             {
                 { "MSBuildExtensionsPath", AppContext.BaseDirectory },
-                { "CscToolExe", GetRunCscPath() },
-                { "VbcToolExe", GetRunVbcPath() },
-                { "MSBuildSDKsPath", GetMSBuildSDKsPath() }
+                { "CscToolPath", GetRunToolPath() },
```

Comment:
For reference, you can see both being [used here](https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/MSBuildTask/Microsoft.CSharp.Core.targets#L141), and [this test](https://github.com/Microsoft/msbuild/blob/06323ab096b9efc505c4266f3228a10b9efdac79/src/Utilities.UnitTests/ToolTask_Tests.cs#L366) that shows ToolExe should just be the filename.